__tests__/app.test.js

```diff
@@ -171,3 +171,37 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe.only("GET /api/articles", () => {
+  test("200: responds with an array of article objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeInstanceOf(Array);
+        articles.forEach((article) => {
```

Comment:
Make sure to assert the length of the array in this test. If the array is empty the test will still pass as we will never git into the body of the function passed into the forEach and as such never make the assertions inside it.